frontend/packages/kubevirt-plugin/src/components/vms/vm-users.tsx

```diff
@@ -0,0 +1,121 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+import { StatusItem } from '@console/shared/src/components/dashboard/status-card/AlertItem';
+import { BlueInfoCircleIcon } from '@console/shared/src/components/status';
+import { fromNow } from '@console/internal/components/utils/datetime';
+import { getName, getNamespace } from '@console/shared/src/selectors/common';
+import { Table, TableRow, TableData } from '@console/internal/components/factory';
+import {
+  withDashboardResources,
+  DashboardItemProps,
+} from '@console/internal/components/dashboard/with-dashboard-resources';
+import { VMIKind } from '../../types';
+import './vm-resource.scss';
+import { VMStatusBundle } from '../../statuses/vm/types';
+import { VMStatus } from '../../constants/vm/vm-status';
+import { Timestamp } from '@console/internal/components/utils';
+import { isGuestAgentInstalled } from '../dashboards-page/vm-dashboard/vm-alerts';
+
+const tableColumnClasses = [
+  classNames('col-lg-3', 'col-md-3', 'col-sm-4', 'col-xs-6'),
+  classNames('col-lg-3', 'col-md-3', 'col-sm-4', 'hidden-xs'),
```

Comment:
> What is the domain? 
domain is the ip address user logged in from, if we need to choose logged in time and domain, I'll choose logged in time

> Shouldn't we switch the priority and show the Login time instead of Elapsed login time on smaller devices? Especially, because there is timezone in Login time

make sense to me too, switching

p.s.
@matthewcarleton in the designs do we have a way to specify how things look on small screens ?

Reply:
done

Reply:
> domain is the ip address user logged in from, if we need to choose logged in time and domain, I'll choose logged in time

I see, I vote for having `user | domain | login time` visible every time

Reply:
ok :-) fixing

Reply:
done

Reply:
> > What is the domain?
> 
> domain is the ip address user logged in from, if we need to choose logged in time and domain, I'll choose logged in time
> 
> > Shouldn't we switch the priority and show the Login time instead of Elapsed login time on smaller devices? Especially, because there is timezone in Login time
> 
> make sense to me too, switching
> 
> p.s.
> @matthewcarleton in the designs do we have a way to specify how things look on small screens ?

If we are using a PF4 component it should be good for mobile right? Is there anything unique about this that doesn't just use a default PF table?

Reply:
> If we are using a PF4 component it should be good for mobile right? Is there anything unique about this that doesn't just use a default PF table?

@matthewcarleton 
yes, the question here is witch columns should magically disappear on small displays.
The discussion above is: should we show 3 or only 2 columns on extra small display.

for example we decided here to show 3 columns ( removed the elapsed time ):
![screenshot-localhost_9000-2020 06 09-21_21_40](https://user-images.githubusercontent.com/2181522/84185231-5c16c080-aa97-11ea-8806-994303d0bcc6.png)